src/BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/GambatteLink.IEmulator.cs

```diff
@@ -23,7 +23,11 @@ public bool FrameAdvance(IController controller, bool render, bool rendersound =
 				{
 					for (int i = 0; i < _numCores; i++)
 					{
-						if (s.Contains($"P{i + 1} "))
+						if (s.Contains($"P{i + 1} - C"))
+						{
+							_linkedConts[i].Set(s.Replace($"P{i + 1} - C", "P"));
+						}
+						else if (s.Contains($"P{i + 1} "))

```

Comment:
I've thought this over more since reviewing the old PR and I'm now firmly against it, as I believe it will break a lot of scripts and possibly parts of EmuHawk.